tests/playwright/pages/banner.spec.ts

```diff
@@ -1,11 +1,27 @@
 import { stateUserAuthPath } from "../utils";
 import { expect, test } from "../utils/fixtures/base";
 import BannerPage from "../utils/pageObjects/banner.page";
+import { deleteAllBanners, postBanner } from "../utils/requests";
+import { testBanner } from "../utils/consts";
+
+const testBannerAPI = {
+  title: testBanner.title,
+  description: testBanner.description,
+  startDate: new Date(testBanner.startDate).getTime(),
+  endDate: new Date(testBanner.endDate).getTime(),
+};
 
 test.describe("admin user banner page", () => {
   test.beforeEach(async ({ bannerPage }) => {
+    try {
+      await deleteAllBanners(bannerPage.page.context());
+    } catch (error) {
+      console.log(
+        `⚠️ API banner deletion failed, falling back to UI: ${error.message}`
+      );
+      await bannerPage.deleteExistingBanners();
```

Comment:
How come there's two methods to delete banners and one is in the catch block?  And what if the one in the catch block throws an error?